.github/workflows/ddd-validation.yml

```
@@ -0,0 +1,222 @@
+        # Check if any role files were modified
+        CHANGED_ROLES=$(git diff --name-only origin/main..HEAD | grep "docs/development/roles/" || echo "")
+        
+        if [[ -n "$CHANGED_ROLES" ]]; then
+          echo "📋 Role files modified in PR:"
+          echo "$CHANGED_ROLES"
+          
+          echo "🔍 Validating modified role files..."
+          cd scripts
+          go build validate-framework.go
+          ./validate-framework ..
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)